src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/tests/__snapshots__/RadioGroup.test.js.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The component should render 1`] = `
+<div
```

Comment:
Then it maybe shouldn't be mocked, or the mock should return something. This way you can't even say if the children of the `RadioGroup` are rendered correctly.